environment-dev.yml

```diff
@@ -25,7 +25,7 @@ dependencies:
   - mypy
   - natsort
   - numba
-  - numpy
+  - numpy==1.24.4
```

Comment:
Was this change because of NumPy 2.0? If so, I think we should pin to `numpy<2.0` instead.

Reply:
Numba requires less than 1.25. I assume this matters only for development?